File: database/model/hub.py

from sqlalchemy import Column,String, Double, Integer
from sqlalchemy.orm import relationship
from database.model.base import Base
class Hub(Base) :
    
    __tablename__ = 'hub' # 테이블명
    __table_args__ = {'schema': 'luckydepot'} # 스키마명 

    id = Column(Integer, primary_key=True, autoincrement=True) 
    name = Column(String)
    lat = Column(Double)
    lng = Column(Double)
    manager = Column(String)
    phone = Column(String)
    mail = Column(String)
    start_time = Column(String)
    end_time = Column(String)
    
    deliver = relationship("Deliver", back_populates="hub")
    management = relationship("Management", back_populates="hub")

File: lib/model/hub.dart
class Hub {
      final int id;
      final String name;
      final String manager;
      final String phone;
      final double lat;
      final double lng;
      final String mail;
      final String start_time;
      final String end_time;


      Hub ({
        required this.id,
        required this.name,
        required this.manager,
        required this.phone,
        required this.lat,
        required this.lng,
        required this.mail,
        required this.start_time,
        required this.end_time,
      });

    Hub.fromMap(Map<String, dynamic> res)
      : id = res['id'],
        name = res['name'],
        manager = res['manager'],
        phone = res['phone'],
        lat = res['lat'],
        lng = res['lng'],
        mail = res['mail'],
        start_time = res['start_time'],
        end_time = res['end_time'];


}